SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -409,6 +433,22 @@ int64 USpatialActorChannel::ReplicateActor()
 	UpdateChangelistHistory(ActorReplicator->RepState);
 
 	ActorReplicator->RepState->LastChangelistIndex = ChangelistState->HistoryEnd;
+	ActorReplicator->RepState->OpenAckedCalled = true;
+
+	// This would indicate we need to flush our state before we could consider going dormant. In Spatial, this
+	// dormancy can occur immediately (because we don't require acking), which means that dormancy can be thrashed
+	// on and off if AActor::FlushNetDormancy is being called (possibly because replicated properties are being updated
+	// within blueprints which invokes this call). Give a few frames before allowing channel to go dormant.
+	if (ActorReplicator->bLastUpdateEmpty == 0)
```

Comment:
== false? (and true below)

Reply:
Curious about the thrashing behaviour.  Is it something you've seen in practice?  Is it any less likely that an update will happen on frame N+3 vs. frame N+1?

Reply:
What i'm wondering is should we have a much longer wait than 2 replication attempts with no update before going dormant?

Reply:
This was occurring in my test scenario where I was updating a replicated float every frame on a dormant actor. That update was forcing the dormancy state to be flushed every frame and essentially meant the actor never went dormant. In spatial the actor flip-flopped it's dormancy, and resulted in lots of dormancy add/remove ops being processed. Three replicated frames was enough to stop the thrashing in my example, but it'll could require further investigation.

Reply:
Oh - and set to 0/1 as despite the name it's actually an 1 bit int under the hood. Following engine usage.